test/GridapEmbeddedTests/TraceFEMTests.jl

```diff
@@ -0,0 +1,64 @@
+module TraceFEMTests
+
+using Gridap
```

Comment:
This file is not being called by the CI

you need to include it in `test/GridapEmbeddedTests/runtests.jl`

Reply:
@fverdugo  done